code/_onclick/telekinesis.dm

```diff
@@ -139,6 +139,12 @@
 
 
 	else
+		if(focus.buckled_mobs)
+			to_chat(user, "<span class='notice'>This object is too heavy to move with something buckled to it!</span>")
+			return
+		for(var/mob/M in focus.contents)
+			to_chat(user, "<span class='notice'>This object is too heavy to move with something inside of it!</span>")
+			return
```

Comment:
I feel like there should be a better way to do this, but I also aint sure off the top of my head.

Reply:
Yeah, this is the best I could figure out